src/ParseQuery.ts

```diff
@@ -1585,6 +1589,49 @@ class ParseQuery<T extends ParseObject = ParseObject> {
     return this._addCondition(key, '$text', { $search: fullOptions });
   }
 
+  /*
+   * Triggers a MongoDb Atlas Text Search
+   *
+   * @param {string} value The string to search
+   * @param {string[]} path The fields to search
+   * @param {object} options (Optional)
+   * @param {string} options.index The index to search
+   * @returns {Promise} Returns a promise that will be resolved with the results
+   * of the search
+   *
+   */
+
+  async search(value: string, path: string[], options?: SearchOptions): Promise<T[]> {
```

Comment:
For consistency we should rename `value` to `query` and it should allow for string | string[]. This should be good enough for a simple search. We can add regex, allowAnalyzedField, and score options later or now if you prefer. For more advanced searching like compound the developer would have to use `query.aggregate`.